Sources/Core/Byte+Alphabet.swift

```diff
@@ -0,0 +1,178 @@
+
+// MARK: Alphabet
+
+extension Byte {
+    /// Returns whether or not the given byte is an arabic letter
+    public var isLetter: Bool {
+        return (.a ... .z).contains(self) || (.A ... .Z).contains(self)
```

Comment:
No idea why that says arabic (copied that code). It should really just say "returns true if between lowercase or uppercase a-z in ascii"